type-annotator-scanner/README.md

```diff
@@ -0,0 +1 @@
+## Type Annotator Scanner (Scanner)
```

Comment:
At some point we might want proper documentation here. Even now, I wouldn't mind a single-paragraph description of what the Type Annotator Scanner does and what it is used for here 🙂 (don't spend a lot of time on it, but it's certainly worth 5-10 minutes to explain its overall purpose)

Reply:
Sure, I added an overview of the module. Will complete it later :)
048a3bb4e86ee5417677840774efb91dda4f5c70